src/components/SavedProjectPlan/SavedProjectPlan.jsx

```diff
@@ -1,133 +1,19 @@
-import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
-import { Box, Button, Flex, Heading, HStack, VStack } from "@chakra-ui/react";
-import { PDFDownloadLink } from "@react-pdf/renderer";
-import { useEffect, useState } from "react";
-import PlanDocumentPDF from "../../components/PlanDocumentPDF/PlanDocumentPDF";
-import StandardCard from "../StandardCard";
-
-const SavedProjectPlans = ({ plan }) => {
-  const [hasLoaded, setHasLoaded] = useState(false);
-  const [showBackArrow, setShowBackArrow] = useState(false);
-  const [showNextArrow, setShowNextArrow] = useState(false);
-  const [currentCardGroup, setCurrentCardGroup] = useState([]);
-  const [currentGroupIndex, setCurrentGroupIndex] = useState(0);
-
-  useEffect(() => {
-    setHasLoaded(true);
-
-    if (plan.cards.length <= 3) {
-      setCurrentCardGroup(plan.cards);
-    } else {
-      setCurrentCardGroup(plan.cards.slice(0, 3));
-      setShowNextArrow(true);
-    }
-  }, [plan.cards]);
-
-  function getNextCardGroup() {
-    const nextGroupIndex = currentGroupIndex + 3;
-
-    setShowBackArrow(true);
-    setCurrentGroupIndex(nextGroupIndex);
-    if (nextGroupIndex + 3 >= plan.cards.length) {
-      setCurrentCardGroup(plan.cards.slice(nextGroupIndex, plan.cards.length));
-      setShowNextArrow(false);
-    } else {
-      setCurrentCardGroup(plan.cards.slice(nextGroupIndex, nextGroupIndex + 3));
-    }
-  }
-
-  function getPreviousCardGroup() {
-    const prevGroupIndex = currentGroupIndex - 3;
-
-    setShowNextArrow(true);
-    setCurrentGroupIndex(prevGroupIndex);
-    if (prevGroupIndex - 3 < 0) {
-      setCurrentCardGroup(plan.cards.slice(0, 3));
-      setShowBackArrow(false);
-    } else {
-      setCurrentCardGroup(plan.cards.slice(prevGroupIndex, prevGroupIndex + 3));
-    }
-  }
-
+import { Box } from "@chakra-ui/react";
```

Comment:
You actually don't need this component. This feels like a page. Just add whatever you have here to the `saved-project-plans.jsx` page.

Reply:
Btw i'm not referring to the Box component. I'm referrring to the whole `SavedProjectPlan.jsx` file. I just didn't know where to put the comment lol.